docs/data/routes/docs/nextjs/internationalization/using-i18n/en.md

```diff
@@ -3,4 +3,80 @@ name: using-i18n
 routeTemplate: ./data/component-templates/article.yml
 title: Using Next.js i18n
 ---
-# Using Next.js i18n with JSS
\ No newline at end of file
+# Using Next.js i18n with JSS
```

Comment:
@nickwesselman Should we add something to this doc about Sitecore config for language embedding (and how that ties into our custom `Link` component)? Or do we have that somewhere else already?

Reply:
Ohhh yes I don't think we've noted that anywhere, we should add.

Reply:
@ambrauer @nickwesselman I can't recall how language embedding is related to our Link component...In our Link component we are using `locale={false}`, so expected that locale included in link value. If href is not a route we are rendering default `sitecore-jss-react` Link. Can you help with it?

Reply:
@illiakovalenko The language embedding configuration for Sitecore would need to be set to `asNeeded` or `always` in order for language prefix to be returned from Sitecore internal links (@nickwesselman feel free to correct me on any of this). We can provide an example Sitecore config patch, but I don't think it's necessary. Here's an example, FYI
```
<configuration xmlns:patch="http://www.sitecore.net/xmlconfig/">
  <sitecore>
    <links>
      <urlBuilder>
        <languageEmbedding>always</languageEmbedding>
      </urlBuilder>
    </links>
  </sitecore>
</configuration>
```

This ties into our sitecore-jss-nextjs `Link` component only for the fact that it will handle client-side i18n'ed route transitions for internal links coming from Sitecore.

Reply:
@ambrauer Thank you for explanation! I added paragraph about it